docker/grouper/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM jeanblanchard/java:jdk-7
+MAINTAINER Dennis Roberts <dennis@cyverse.org>
+ARG git_commit=unknown
+LABEL org.iplantc.de.javabase.git-ref="$git_commit"
```

Comment:
It doesn't make much sense to overload this label for the grouper image as well as the javabase image. If we do want to add an extra label to track the git-ref when this image was built, we should give it a new name.